stake-pool/py/tests/test_a_time_sensitive.py

```diff
@@ -51,21 +68,47 @@ async def test_increase_decrease_this_is_very_slow(async_client, validators, pay
     for validator in validator_list.validators:
         assert validator.last_update_epoch != 0
         assert validator.transient_stake_lamports == 0
-        assert validator.active_stake_lamports == increase_amount + minimum_amount
+        assert validator.active_stake_lamports == increase_amount + minimum_amount + stake_rent_exemption
+
+    def decrease(validator, decrease_amount, ephemeral_stake_seed=None):
+        return decrease_validator_stake(async_client, payer, payer, stake_pool_address, validator,
+                                        decrease_amount, ephemeral_stake_seed=ephemeral_stake_seed)
 
     # decrease from all
     futures = [
-        decrease_validator_stake(async_client, payer, payer, stake_pool_address, validator, decrease_amount)
+        decrease(validator, decrease_amount // 2)
         for validator in validators
     ]
     await asyncio.gather(*futures)
 
+    # validate the decrease is now on the transient account
+    resp = await async_client.get_account_info(validator_list_address, commitment=Confirmed)
+    data = resp['result']['value']['data']
+    validator_list = ValidatorList.decode(data[0], data[1])
+    for validator in validator_list.validators:
+        assert validator.transient_stake_lamports == decrease_amount // 2 + stake_rent_exemption
+        assert validator.active_stake_lamports == increase_amount-decrease_amount//2+minimum_amount+stake_rent_exemption
+
+    print("Waiting for epoch to roll over")
+    await waiter.wait_for_next_epoch(async_client)
+    await update_stake_pool(async_client, payer, stake_pool_address)
```

Comment:
Is this the part that should be removed in order to make sure that the decrease additional bit is actually happening?

Reply:
If I remove that then the call to DecreaseAdditionalValidatorStake fails with the following message:

'...Program Stake11111111111111111111111111111111111111 invoke [2]', 'Checking if destination stake is mergeable', 'stake account with transient stake cannot be merged ...'

I have also tried changing so that the deactivating stake is < 8% of the total stake, but still get the same error. Will spend some more time debugging to figure out what is going wrong (and use the Rust test as a reference).